app/models/custom_field.rb

```diff
@@ -33,11 +33,14 @@
 class CustomField < ActiveRecord::Base
   include Translatable
 
+  OVERRIDE_ASSOCIATIONS_OPTIONS = %i(inherit override)
```

Comment:
Using inherit and override will affect loan questions that already have used true/false or other values.